trackpy/tests/test_feature.py

```diff
@@ -428,6 +428,15 @@ def test_subpx_precision(self):
         expected = DataFrame(np.asarray(pos).reshape(1, -1), columns=cols)
         assert_allclose(actual, expected, atol=PRECISION)
 
+    def test_nocharacterize(self):
+        pos = gen_nonoverlapping_locations((200, 300), 9, 5)
+        image = draw_spots((200, 300), pos, 5)
+        f_c = tp.locate(image, 5, engine=self.engine, characterize=True)
+        f_nc = tp.locate(image, 5, engine=self.engine, characterize=False)
+
+        assert len(f_nc.columns) == 3
+        assert_allclose(f_c.values[:3], f_nc.values[:3])
```

Comment:
You need to slice the feature ndarrays on the second axis, now there is a shape mismatch. I would say, don't slice f_nc and do [:, :3] for f_c.